docs/4.0/components/forms.md

```diff
@@ -700,7 +700,7 @@ While Bootstrap will apply these styles in all browsers, Internet Explorer 11 an
 Provide valuable, actionable feedback to your users with HTML5 form validation–[available in all our supported browsers](https://caniuse.com/#feat=form-validation). Choose from the browser default validation feedback, or implement custom messages with our built-in classes and starter JavaScript.
 
 {% callout warning %}
-We **highly recommend** custom validation styles as native browser defaults are not announced to screen readers.
+We **highly recommend** custom validation styles as native browser defaults are not announced to screen readers. Please ensure to manage `aria-describedby` attribute on `input` element according to the validation state.
```

Comment:
i'd add a comma after "validation styles" to make it easier to read.

Reply:
"ensure to" probably better changed to "ensure you" (my preference). Otherwise "make sure to manage" sounds more natural. also good place to include the `aria-invalid` thing in the prose, e.g. "Please ensure you manage the `aria-invalid` and `aria-describedby` attributes on the `input` element..."